src/pages/Clients/Consensus/Prysm.tsx

```diff
@@ -11,9 +12,31 @@ import { Link } from '../../../components/Link';
 import { PRYSM_INSTALLATION_URL } from '../../../utils/envVars';
 import { FormattedMessage, useIntl } from 'react-intl';
 
+const ClientDiversityWarning = styled(Text as any)`
+  background: #ffdeb32e;
+  border: 1px solid burlywood;
+  padding: 30px;
+  border-radius: 4px;
+`;
+
 // eslint-disable-next-line no-unused-vars
 export const PrysmDetails = ({ shortened }: { shortened?: boolean }) => (
   <>
+    <ClientDiversityWarning>
+      <FormattedMessage
+        defaultMessage="Currently the majority of validators run Prysm as their consensus client.
+      Client diversity is extremely important for the network health of Ethereum:
+      A bug in a client with a share of over 33% can cause Ethereum to go offline. If the client has
+      a super-majority (>66%), a bug could cause the chain to incorrectly split, potentially leading to
+      slashing."
+      />
+      <Link
+        to="https://ethereum.org/en/developers/docs/nodes-and-clients/client-diversity/"
+        primary
+      >
+        <FormattedMessage defaultMessage="More on client diversity" />
+      </Link>
```

Comment:
I like this ❤️... two small notes, "supermajority" doesn't need to be hyphenated, and I just suggested adding one last callout encouraging people to chose another client. Feel free to reword or ignore, just my thoughts. 

```suggestion
      <p>
        <FormattedMessage
          defaultMessage="Currently the majority of validators run Prysm as their consensus client.
            Client diversity is extremely important for the network health of Ethereum:
            A bug in a client with a share of over 33% can cause Ethereum to go offline. If the client has
            a supermajority (>66%), a bug could cause the chain to incorrectly split, potentially leading to
            slashing."
        />
      </p>
      <p>
        <FormattedMessage
          defaultMessage="If at all possible, consider running another client at this time to help protect yourself and the network."
        />
      </p>
      <p>
        <Link
          to="https://ethereum.org/en/developers/docs/nodes-and-clients/client-diversity/"
          primary
        >
          <FormattedMessage defaultMessage="More on client diversity" />
        </Link>
      </p>
```